clap_complete_fig/tests/common.rs

```diff
@@ -46,11 +46,18 @@ pub fn special_commands_command(name: &'static str) -> clap::Command<'static> {
                 .arg(
                     clap::Arg::new("config")
                         .long("--config")
+                        .hide(true)
```

Comment:
This file is kept in sync with the same in `clap_complete`.  Please update it there as well and update the snapshots.  You are not responsible for any other updates in the other completion engines.

Reply:
Looks like snapshots weren't updated for `clap_complete`.

Reply:
I've seen, I guess they don't get updated when I run the overwrite command from the root of the project.

Reply:
... odd.  They should.

Reply:
Tried to reproduce the problem locally but not seeing it.